src/backend/db/database_manager.py

```diff
@@ -484,9 +484,27 @@ def create_ticket(self, customer_guid, chat_id, title, description, priority, cu
                 )
 
                 # Fetch the generated ticket_id
-                result = session.execute("SELECT LAST_INSERT_ID()").fetchone()
-                ticket_id = result[0]
-                logger.debug(f"Created ticket with ID: {ticket_id}")
+                result = session.execute(
```

Comment:
Instead of querying again you can use the RETURNING * to return the inserted row

This will give you the correct Ticket_id

INSERT INTO tickets (chat_id, title, description, priority)
VALUES (:chat_id, :title, :description, :priority)
RETURNING *;

Use the RETURNING keyword while enabling a new row and get the ticket_id from the returned values

                # Insert into tickets table
                query = '''INSERT INTO tickets (chat_id, title, description, priority)
                           VALUES (:chat_id, :title, :description, :priority) 
                            RETURNING *'''

Reply:
Sir, I got a syntax error when I used the RETURNING clause because PostgreSQL supports the RETURNING clause, but MySQL does not